x/superfluid/keeper/twap_price.go

```diff
@@ -40,8 +40,8 @@ func (k Keeper) GetSuperfluidOSMOTokens(ctx sdk.Context, denom string, amount sd
 	}
 
 	decAmt := multiplier.Mul(amount.ToDec())
-	asset := k.GetSuperfluidAsset(ctx, denom)
-	return k.GetRiskAdjustedOsmoValue(ctx, asset, decAmt.RoundInt())
+	k.GetSuperfluidAsset(ctx, denom)
+	return k.GetRiskAdjustedOsmoValue(ctx, decAmt.RoundInt())
```

Comment:
```suggestion
	return k.GetRiskAdjustedOsmoValue(ctx, decAmt.RoundInt())
```

Reply:
GetSuperfluidAsset panics on denom not existing, which is why it is being called here